### DISCUSSION TITLE
Confused about contents of `.github` directory in this repo

### DISCUSSION THREAD

**Discussion by SethTisue** [sbt/sbt-ci-release#324] [Q&A]
```
% tree .github
.github
├── dependabot.yml
├── release-drafter.yml
└── workflows
    ├── ci.yml
    ├── release-drafter.yml
    └── release.yml
```

why are there two `release-drafter.yml` files?

why is `dependabot.yml` not under `workflows`?

**Comment by eed3si9n** [comment]
Re: `dependabot.yml` - https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates

> You enable Dependabot version updates by checking a `dependabot.yml` configuration file in to your repository's `.github` directory.

In Scala ecosystem, this is used to bump the versions of the GitHub Actions.

**Comment by SethTisue** [comment]
as for where release-drafter goes, the repos on my hard drive are about equally divided:

```
 ls */.github/release-drafter.y* | wc -l
      25
% ls */.github/workflows/release-drafter.y* | wc -l
      23
```

**Comment by mkurz** [comment]
> why are there two `release-drafter.yml` files?

One is the GitHub actions workflow: https://github.com/sbt/sbt-ci-release/blob/main/.github/workflows/release-drafter.yml
The other one is the default release drafter config file: https://github.com/sbt/sbt-ci-release/blob/main/.github/release-drafter.yml
See [docs](https://github.com/release-drafter/release-drafter?tab=readme-ov-file#configuration):
> Once you’ve added Release Drafter to your repository, it must be enabled by adding a `.github/release-drafter.yml` configuration file to each repository. The configuration file **must** reside in your default branch, no other configurations will be accepted.

You can change the name/location of the release drafter config file in the workflow with:
```hocon
config-name: my-release-drafter-config.yml
# or in a sub-folder of .github
config-name: release-drafts/my-release-drafter-config.yml
```

**Comment by SethTisue** [comment]
thanks guys
